src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1392,8 +1388,19 @@ private async Task AttachToTarget(SessionId sessionId, CancellationToken token)
             // see https://github.com/mono/mono/issues/19549 for background
             if (sessions.Add(sessionId))
             {
-                string checkUncaughtExceptions = $"throw \"{sPauseOnUncaught}\";";
-                string checkCaughtExceptions = $"try {{throw \"{sPauseOnCaught}\";}} catch {{}}";
+                string checkUncaughtExceptions = "";
+                string checkCaughtExceptions = "";
+
+                //we only need this check if it's a non-vs debugging
+                if (sessionId == SessionId.Null)
```

Comment:
How is this checking for non-vs debugging?
And does `SessionId.Null` represent a default context, which might have an entry in `contexts` dictionary?

Reply:
As I could see by debugging, when I'm debugging from VS SessionId is not null, when I'm debugging from chrome SessionId is null.
Maybe @lewing knows how to explain better.

Reply:
I think, the non-null sessionId means that it is a "flat access".
IIUC, chrome devtools will move to it too, so this might not be a good test. I'll defer to @lewing .